Modules/LearningSequence/classes/LSItems/LSItemInformation.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * @author Daniel Weise <daniel.weise@concepts-and-training.de>
```

Comment:
den author-Teil bitte in die Klassenbeschreibung.

Reply:
ja